repository/Grease-Core.package/GRPlatform.class/instance/asMethodReturningString.named..st

```diff
@@ -0,0 +1,21 @@
+file library
+asMethodReturningString: aByteArrayOrString named: aSymbol
```

Comment:
I decided to add a default implementation that works for anybody that does not need special behaviour.

Reply:
Could we also include a test for this? It will also verify the correct behaviour in all tested platforms in the github ci.

Reply:
> Could we also include a test for this?

I can try. The trick is to figure out whether the dialect supports Unicode.